torchaudio/functional/filtering.py

```diff
@@ -11,6 +11,12 @@ def _dB2Linear(x: float) -> float:
     return math.exp(x * math.log(10) / 20.0)
 
 
+def _float2Tensor(x: float, dtype: torch.dtype, device: torch.device):
```

Comment:
This is very similar to [torch.as_tensor](https://pytorch.org/docs/master/generated/torch.as_tensor.html#torch.as_tensor). For example

```
>>> t = torch.as_tensor(4, dtype=torch.float)
>>> t
tensor(4.)
>>> t.dtype
torch.float32
```